clang/lib/Frontend/InitPreprocessor.cpp

```diff
@@ -1179,8 +1179,14 @@ static void InitializePredefinedMacros(const TargetInfo &TI,
     Builder.defineMacro("__SYCL_DEVICE_ONLY__", "1");
     Builder.defineMacro("SYCL_EXTERNAL", "__attribute__((sycl_device))");
 
-    if (TI.getTriple().isNVPTX()) {
-        Builder.defineMacro("__SYCL_NVPTX__", "1");
+    const llvm::Triple &DeviceTriple = TI.getTriple();
+    if (DeviceTriple.isNVPTX()) {
+      Builder.defineMacro("__SYCL_NVPTX__", "1");
+    }
```

Comment:
```suggestion
    if (DeviceTriple.isNVPTX())
      Builder.defineMacro("__SYCL_NVPTX__");
```
Style nit and `1` is already the default argument anyway.

Reply:
Whoops, literal removal was an accidental change. As for the style, `clang-format` insists that the padding be 2 characters instead of 4 - I'd have to stick to that in sight of CI checks.

Reply:
Sorry, the style bit I meant was removing the curly braces around a single-line substatement. That's a community style thing.

Reply:
Edit: I didn't realize that removing `1` was a suggestion and not something I've already done. Would it be OK if I removed the literal in a separate PR so as not to mix the unrelated purposes?

Reply:
I'm fine either way! I see it as related because we're already touching this code for an NFC cleanup, so more NFC cleanup on the same line seems reasonable.

Reply:
Styling issue addressed in https://github.com/intel/llvm/pull/3869/commits/b93a820cf1c3005a3d479dcb810ce553586f2a6d. As for the default value, I already have a local patch for LIT-testing the `__SYCL_NVPTX__` macro setting (in scope of #3972), so I'll apply the suggestion in that PR.

Reply:
Sounds great, thanks!